ucm2/Qualcomm/sdm845/HiFi-MM1.conf

```diff
@@ -0,0 +1,77 @@
+# Use case configuration for LenovoYOGAC6301.
+# Author: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
+
+SectionVerb {
+
+	EnableSequence [
+		cset "name='SLIMBUS_0_RX Audio Mixer MultiMedia1' 1"
+		cset "name='SLIMBUS_1_RX Audio Mixer MultiMedia3' 1"
+		cset "name='MultiMedia2 Mixer SLIMBUS_0_TX' 1"
+	]
+
+	Include.wcde.File "/codecs/wcd934x/DefaultEnableSeq.conf"
+	Include.wsae.File "/codecs/wsa881x/DefaultEnableSeq.conf"
+
+	Include.wcdd {
+		File "/codecs/wcd934x/DefaultDisableSeq.conf"
+		Before.DisableSequence "0"
+	}
+
+	DisableSequence [
+		cset "name='SLIMBUS_0_RX Audio Mixer MultiMedia1' 0"
+		cset "name='SLIMBUS_1_RX Audio Mixer MultiMedia3' 0"
+		cset "name='MultiMedia2 Mixer SLIMBUS_0_TX' 0"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker playback"
+	
+	Include.wcdspke.File "/codecs/wcd934x/SpeakerEnableSeq.conf"
+	Include.wcdspkd.File "/codecs/wcd934x/SpeakerDisableSeq.conf"
+	Include.wsaspke.File "/codecs/wsa881x/SpeakerEnableSeq.conf"
+	Include.wsaspkd.File "/codecs/wsa881x/SpeakerDisableSeq.conf"
+	
+
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "hw:${CardId},0"
+		PlaybackMixer "default:${CardId}"
+		PlaybackMixerElem "Speaker Digital"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Headphones playback"
+
+	Include.wcdhpe.File "/codecs/wcd934x/HeadphoneEnableSeq.conf"
+	Include.wcdhpd.File "/codecs/wcd934x/HeadphoneDisableSeq.conf"
+
+	Value {
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId},2"
+		PlaybackMixer "default:${CardId}"
+		PlaybackMixerElem "HP Digital"
+	}
+}
+
+SectionDevice."Mic" {
+	Comment "Mic"
+
+	Include.wcdmice.File "/codecs/wcd934x/HeadphoneMicEnableSeq.conf"
+	Include.wcdmicd.File "/codecs/wcd934x/HeadphoneMicDisableSeq.conf"
+
+	DisableSequence [
+		cset "name='AMIC MUX0' ZERO"
+	]
+	
+	Value {
+		CapturePriority 100
+		CapturePCM "hw:${CardId},1"
+		CaptureVolume "ADC2 Volume"
```

Comment:
This should be probably `CaptureMixerElem "ADC2"` ?

Reply:
Thanks Jaroslav,
both CaptureVolume and CaptureMixerElem seems to work for me, any particular reason why CaptureMixerElem is prefered over CaptureVolume?

Reply:
Pulseaudio / pipewire use only ...MixerElem values. The direct CTL interface is ignored. PA use only software mute then (you may verify, if ADC2 volume changes in ALSA mixer when you use PA). I should describe it more in https://www.alsa-project.org/alsa-doc/alsa-lib/group__ucm__conf.html .

Reply:
You can define both values, of course.

Reply:
Thanks, I will update the patch with CaptureMixerElem and not use both to avoid any confusion.